dapp/src/components/search-component/InputWrapper.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { SecondaryText } from '@iota/apps-ui-kit';
+import cx from 'clsx';
+
+export interface InputWrapperProps {
+    errorMessage?: string;
+    disabled?: boolean;
+}
+
+export function InputWrapper({
+    disabled,
+    errorMessage,
+    children,
+}: React.PropsWithChildren<InputWrapperProps>) {
+    return (
+        <div
+            className={cx('group flex w-full flex-col gap-y-2', {
+                'cursor-not-allowed opacity-40': disabled,
+                errored: errorMessage,
+                enabled: !disabled,
+            })}
+        >
+            {children}
+            {errorMessage && (
+                <div
+                    className={cx(
+                        'flex flex-row items-center',
+                        errorMessage ? 'justify-between gap-md' : 'justify-end',
+                    )}
+                >
+                    {errorMessage && <SecondaryText hasErrorStyles>{errorMessage}</SecondaryText>}
+                </div>
+            )}
+        </div>
```

Comment:
The if conditions inside are not needed, maybe just remove them no? Or you wanted to do something else?
```suggestion
            {errorMessage && (
                <div
                    className={cx(
                        'flex flex-row items-center',
                        errorMessage ? 'justify-between gap-md' : 'justify-end',
                    )}
                >
                    {errorMessage && <SecondaryText hasErrorStyles>{errorMessage}</SecondaryText>}
                </div>
            )}
```